commit 462e4e5c544bcc4f30337f9004e29d9a68e62e5c (refs/changes/88/3888/2)
Author: Joshua M. Clulow <jmc@joyent.com>
Date:   2018-04-30T21:10:05+00:00 (1 year, 5 months ago)
    
    MORAY-468 "close-handles" test does not account for stdout/stderr
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/test/close-handles.test.js b/test/close-handles.test.js
index 7cf2984..8e5f247 100644
--- a/test/close-handles.test.js
+++ b/test/close-handles.test.js
@@ -66,7 +66,9 @@ tape.test('client close actually closes sockets', function (t) {
         }
     ], function (err) {
         t.error(err);
-        t.equal(0, process._getActiveHandles().length);
+        t.deepEqual([], process._getActiveHandles().filter(function (h) {
+            return (h !== process.stderr && h !== process.stdout);
+        }));
         t.end();
     });
 });
